R/fit_mmrm.R

```diff
@@ -98,7 +98,7 @@ h_get_diagnostics <- function(fit) {
 #'
 #' @export
 #'
-#' @examples
+#' @examplesIf requireNamespace("TMB")
```

Comment:
It looks like the standard applied in other packages is `require()`. Both are fully functional and this is just for consistency

```suggestion
#' @examplesIf require("TMB", quietly = TRUE)
```